packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -123,7 +123,7 @@
       &:checked:disabled,
       &:checked[aria-disabled="true"] {
         @include format-input {
-          @include add-background-svg($input-checkmark);
+          @include add-background-svg("correct8");
```

Comment:
I noticed the checkbox component still shows some visual regressions when I set the body background to color grades 40+ (I tested cyan).

![image](https://github.com/uswds/uswds/assets/93996430/f71088ca-4eee-437b-8dc3-20086f7001f4)

I suspect it is because it is not using `$input-checkmark` here. 

```suggestion
          @include add-background-svg($input-checkmark);
```

Reply:
Fixed in 3acdf36. All styles for `checkbox-and-radio-colors.scss` are reverted to what's currently in develop for 3.4.1.